### ISSUE TITLE
IntelliJ doesn't show <Click to see difference> link with Groovy 2.5

### ISSUE THREAD

**Issue by huylv** [assertj/assertj#1364]
#### Summary
IntelliJ 2018 doesn't show `<Click to see difference>` link if Groovy version is 2.5. It works with Groovy 2.4. I have tested on Redhat and macOS. I'm not sure if this is caused by assertj or not so my apologies  if I report to the wrong project.

#### Example
```groovy
// TestSomething.groovy
assertThat(1).isEqualTo(2)

// build.gradle
dependencies {
    compile group: 'org.codehaus.groovy', name: 'groovy-all', version: '2.4.11'

    testCompile group: 'junit', name: 'junit', version: '4.12'
    testCompile group: 'org.assertj', name: 'assertj-core', version: '3.11.1'
}
```

Output:
```
org.junit.ComparisonFailure: 
Expected :2
Actual   :1
 <Click to see difference>
```

if I change groovy version to `2.5.2`, the output would be:
```
org.opentest4j.AssertionFailedError: 
Expecting:
 <1>
to be equal to:
 <2>
but was not.
```

**Comment by joel-costigliola** [comment]
`<Click to see difference>` comes from IDE support for the thrown `org.junit.ComparisonFailure` or `org.opentest4j.AssertionFailedError`.
I'm pretty sure both are supported in recent version of Idea (what version do you use ?), I use eclipse and they are supported.

There is nothing AssertJ can do since it is an IDE feature.

I'm closing this issue since it is not related to AssertJ but feel free to continue the conversation.

**Comment by huylv** [comment]
I'm using IntelliJ version 2018.2.3, released September 2018. It's weird that my friend could see the link in his centOS machine, running the same IntelliJ version, but I can't make it work on my macbook and Redhat 7 machine.

**Comment by joel-costigliola** [comment]
Weird indeed, I'm sure 2018.2.3 supports `org.opentest4j.AssertionFailedError` difference and that it should not be OS specific

I'm a linux guy, I can't help you much with macos stuff.

**Comment by huylv** [comment]
Thank you, Joel. Probably I'll need to look for help somewhere else.

**Comment by filiphr** [comment]
This is a bug in IntelliJ. See [IDEA-179482](https://youtrack.jetbrains.com/issue/IDEA-179482). If you have JUnit 4 and opentest4j present it is not using the nice click to see differences even though it should. This should be fixed in 2019.1. The EAP for it would start soon
